images/Dockerfile.opensuse-leap

```diff
@@ -1,8 +1,16 @@
+# syntax=docker/dockerfile:1.4
 ARG BASE_IMAGE=opensuse/leap:15.4
 
 FROM $BASE_IMAGE
 
-RUN zypper in -y \
+ARG K3S_VERSION
```

Comment:
This was dropped already from all the docker files as it's not used anywhere:)

Reply:
ah that was probably just a bad rebase thanks!